### ISSUE TITLE
Retry documentation update/install on error

### ISSUE THREAD

**Issue by hanxue** [egoist/devdocs-desktop#64]
I am on a flaky connection, and downloading the docs offline usually result in this

![image](https://user-images.githubusercontent.com/760008/35468934-71b36696-0364-11e8-8d7c-edc45b75ba8a.png)

I will have to close the Preferences window and re-open, then install or update again. 

Can there be an automated retry, or a retry button for those docs that fail to download?

**Comment by egoist** [comment]
Maybe this could be imporved on @Thibaut 's side ?

**Comment by Thibaut** [comment]
Good idea (should add a "Retry" link). Please open an issue https://github.com/Thibaut/devdocs.

**Comment by hanxue** [comment]
OK, I have raised https://github.com/Thibaut/devdocs/issues/753
